contracts/protocol/PositionHouse.sol

```diff
@@ -161,7 +161,7 @@ contract PositionHouse is
             _trader,
             pQuantity,
             _leverage,
-            pResp.exchangedQuoteAssetAmount / _quantity,
+            (pResp.exchangedQuoteAssetAmount * _positionManager.getBasisPoint()) / _quantity,
```

Comment:
event OpenMarket currently emitting the average entry price of market order, calculated by exchangedQuoteAmount / quantity. but this formula will return an integer number, which can be wrong when the average price is a decimal number. the solution is changes from price => pip